src/main/java/reactor/kafka/receiver/errors/ReceiverExceptionHandler.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package reactor.kafka.receiver.errors;
+
+/**
+ * Interface that specifies how an exception thrown by Kafka Receiver
+ * (e.g., failing to commit to Kafka) should be handled.
+ */
+public interface ReceiverExceptionHandler {
+
+    /**
+     * Inspect the exception received.
+     * @param exception the actual exception
+     */
+    ReceiverExceptionHandlerResponse handle(final Exception exception);
+
+    /**
+     * Enumeration that describes the response from the exception handler.
+     */
+    enum ReceiverExceptionHandlerResponse {
+        /* continue with processing */
+        CONTINUE(0, "CONTINUE"),
```

Comment:
is there a real use for `name` and `id` here? enums naturally have them... (`name()` and `ordinal()`)

Reply:
I tried to implement the handler as close as possible to [Kafka Streams handlers](https://github.com/axbaretto/kafka/blob/master/streams/src/main/java/org/apache/kafka/streams/errors/DeserializationExceptionHandler.java). Should I change it?

Reply:
I don't think the enum is going to evolve much in terms of name. as far as I can see, you don't use the `id()`, so I'd say you can simplify the enum to one without attributes. the only benefit would have been if the enums were marshalled over the wire for another system/language to unmarshall

Reply:
attributes removed